server/utils/utils.ts

```diff
@@ -789,3 +789,8 @@ export const mapSlots = (createJourney: CreateAnActivityJourney) => {
 export const padNumber = (num: number, length = 2) => {
   return (new Array(length).fill('0').join('') + num).slice(-length)
 }
+
+export const setAttribute = (object: { [key: string]: string }, key: string, value: string) => {
```

Comment:
Nunjucks doesn't provide a way to edit object attributes – this util enables that.